### PR TITLE
feat: Extract or move current evm fee related functions to EVM folder

### PR DESCRIPTION
**Pull request by enemycnt**
## Related Issue Or Context

Closes: #195

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [ ] I have updated the documentation locally and in chainbridge-docs.
- [ ] I have added tests to cover my changes.
- [ ] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

File: packages/sdk/.eslintrc.cjs
Comment (wainola) [inline packages/sdk/.eslintrc.cjs:1]: how this file works with the others in the others pull request?

File: packages/sdk/src/chains/EVM/fee/basicFee.ts [hunk 1]
```diff
@@ -1,13 +1,26 @@
...
 import { ethers } from 'ethers';
-import { FeeDataResult } from '../types';
-import { constructDepositDataEvmSubstrate } from '../utils/helpers';
+import { FeeDataResult } from '../../../types';
+import { constructDepositDataEvmSubstrate } from '../../../utils/helpers';
 
 /**
- * @name calculateBasicfee
- * @description calculates and returns the feeData object after query the FeeOracle service
+ * Calculates and returns the feeData object after query the FeeOracle service
```
Comment (wainola) [inline packages/sdk/src/chains/EVM/fee/basicFee.ts:7]: Could you add the `@name` tag?

File: packages/sdk/src/chains/EVM/fee/basicFee.ts [hunk 2]
```diff
@@ -53,7 +66,7 @@ export const calculateBasicfee = async ({
       type: 'basic',
     };
   } catch (error) {
-    console.error(error);
-    return Promise.reject(new Error('Invalidad basic fee response'));
+    console.error('Invalidad basic fee response');
```
Comment (wainola) [inline packages/sdk/src/chains/EVM/fee/basicFee.ts:69]: Could we log the error anyways?

File: packages/sdk/src/chains/EVM/fee/dynamicFee.ts [hunk 3]
```diff
@@ -2,23 +2,21 @@ import { DynamicERC20FeeHandlerEVM__factory } from '@buildwithsygma/sygma-contra
...
 };
 
 /**
- * @name createOracleFeeData
- * @description creates the fee data to send
- * @param oracleResponse - OracleResource object
- * @param amount - amount to send
- * @param tokenResource - the resource Id of the token
- * @returns {string} - hex of the fee data constructed
+ * Creates feeData structure with the following parameters.
```
Comment (wainola) [inline packages/sdk/src/chains/EVM/fee/dynamicFee.ts:14]: Could you add the `@name` tag?

File: packages/sdk/src/chains/EVM/fee/dynamicFee.ts [hunk 4]
```diff
@@ -61,12 +59,37 @@ export const createOracleFeeData = (oracleResponse: OracleResource, amount: stri
 };
 
 /**
- * @name calculatedFeeData
- * @description calculates the fee data after query the FeeOracle service
- * @param {Object} - object provided to calculate the fee data after query oracle service
- * @returns {Promise<FeeDataResult | undefined>}
+ * Calculates the dynamic fee for a transaction using the provided parameters and the Fee Oracle.
```
Comment (wainola) [inline packages/sdk/src/chains/EVM/fee/dynamicFee.ts:62]: Same as above

File: packages/sdk/src/chains/EVM/fee/dynamicFee.ts [hunk 5]
```diff
@@ -124,10 +147,15 @@ export const calculateFeeData = async ({
 };
 
 /**
- * @name requestFeeFromFeeOracle
- * @description query the FeeOracle service to get the fee
- * @param {Object}
- * @returns {Promise<OracleResponse | undefined>}
+ * Fetches oracle resource data from the FeeOracle service.
```
Comment (wainola) [inline packages/sdk/src/chains/EVM/fee/dynamicFee.ts:150]: Same as above

**Review by wainola — CHANGES_REQUESTED** [review]

File: packages/sdk/.eslintrc.cjs
Comment (enemycnt) [reply packages/sdk/.eslintrc.cjs:1]: It works well for CI, the updated version for VScode will be merged with other PRs. I don't want to copy-paste it to every PR

File: packages/sdk/src/chains/EVM/fee/basicFee.ts (hunk 2 above)
Comment (enemycnt) [reply packages/sdk/src/chains/EVM/fee/basicFee.ts]: done

**Review by MakMuftic — APPROVED** [review]

**Review by wainola — APPROVED** [review]

**Pull request merged**